### ISSUE TITLE
'-' character in HTML attributes

### ISSUE THREAD

**Issue by andreyk0** [ivan-m/graphviz#20]
Hi!
I've been trying to load jar (java) dependency graphs using this library and run into a small problem with '-' character.
Labels end up looking like
```[label=<commons-collections<BR/><B>commons-collections</B><BR/>3.2.1> style=""]```

CLI `dot` executable doesn't seem to object to that but graphviz lib requires '-' to be always escaped.

So, e.g. 
```dot
digraph "dependency-graph" {
    graph[rankdir="LR"]
    edge [
        arrowtail="none"
    ]
    "t1"[label=<foo-bar> style=""]
    "t2"[label=<ha<B>foo-BLAH_2.12</B><BR/>1.0-SNAPSHOT> style=""]

    "t2" -> "t1"
}
```
Parses fine on the command line but not with the library.

If I change this one line then it works:

```diff
diff --git a/Data/GraphViz/Attributes/HTML.hs b/Data/GraphViz/Attributes/HTML.hs
index af59eb9..12eb3e5 100644
--- a/Data/GraphViz/Attributes/HTML.hs
+++ b/Data/GraphViz/Attributes/HTML.hs
@@ -572,7 +572,7 @@ htmlEscapes = [ ('"', "quot")
               , ('>', "gt")
               , ('&', "amp")
               ]
-              ++ map numEscape ['-', '\'']
+              ++ map numEscape ['\'']
   where
     numEscape c = (c, T.pack $ '#' : show (ord c))
 ```

Would it be possible to make this change permanent?
Thanks!

**Comment by ivan-m** [comment]
Looks like I either was over-zealous in my escaping or they've changed the rules:

>  As HTML strings are processed like HTML input, any use of the ", &, <, and > characters in literal text or in attribute values need to be replaced by the corresponding escape sequence. For example, if you want to use & in an href value, this should be represented as &amp;.

Want to make a pull request?

**Comment by ivan-m** [comment]
(Looks like `'` doesn't need to be escaped any more either. )

**Comment by andreyk0** [comment]
@ivan-m Hey! Does this patch work for you or do I need to do more?
Thanks!

**Comment by ivan-m** [comment]
I had wanted to test this against the sample graphs  from Graphviz, but found some existing issues with HTML labels that I'm trying to fix first :/

**Comment by andreyk0** [comment]
Ah, cool! Thanks for looking into it, let me know if I need to do anything else.

**Comment by ivan-m** [comment]
Closed in #21
